ciongke/cmd/test-pr/main.go

```diff
@@ -54,7 +54,7 @@ const (
 
 	// Inputs are context, URL, commit, retest command.
 	// The deletion logic requires that it start with context.
-	bodyFormat = `%s [**failed**](%s) for commit %s.
+	bodyFormat = `%s [**failed**](%s) for commit %s (see full [pr test history](http://ci-test.k8s.io/%s)).
```

Comment:
`%s [**failed**](%s) for commit %s. [Full PR test history](http://ci-test.k8s.io/%s).`

Also, update the comment with the new input.